### PR TITLE
Built front end of login page

### PR DESCRIPTION
**Pull request by FadyAPhilips**
cleaned up code in other files
added temp background
fixed z-index

### CODE REVIEW

File: src/Routes/Login.js [hunk 1]
```diff
@@ -1,51 +1,30 @@
...
-
-const iconStyling = {
-  color: "#c1c1c1",
-  fontSize: "18px",
-  marginLeft: "8px",
-};
-
+import LoginModal from "../UI/LoginModal";
+import BlurDiv from "../UI/BlurDiv";
+import backgroundImage from "../StarsBG.jpg";
```
Comment (amrmohamedmero) [inline src/Routes/Login.js:6]: aint this background a bit pixelated.
is the ratio that you are using working with different screens

File: src/Routes/Login.js [hunk 2]
```diff
@@ -1,51 +1,30 @@
...
-        showMyListIcon={true}
-        showSearchbar={true}
+        username={null}
+        showMyListIcon={false}
+        showSearchbar={false}
         showLoginButton={false}
         showSignUpButton={true}
         showLogOutButton={false}
       />
-      <Modal closeOnClick={() => console.log("object")}>
```
Comment (amrmohamedmero) [inline src/Routes/Login.js:25]: is it important to have this line here 
**closeOnClick={() => console.log("object")}**

File: src/Routes/Login.js (hunk 1 above)
Comment (FadyAPhilips) [reply src/Routes/Login.js:6]: It is blurred on purpose. We can change the blurry-ness with a prop. the actual image is in 4k so it should work on most displays.

File: src/Routes/Login.js (hunk 2 above)
Comment (FadyAPhilips) [reply src/Routes/Login.js:25]: it isn't there. that is one of the lines that were removed

**Review by markarmanus — APPROVED** [review]

**Pull request merged**